columnflow/plotting/plot_util.py

```diff
@@ -112,17 +196,23 @@ def get_stack_integral() -> float:
             )
         return stack_integral
 
-    # apply "scale" setting directly to the hists
     for proc_inst, h in hists.items():
+        # apply "scale" setting directly to the hists
         scale_factor = getattr(proc_inst, "scale", None) or proc_inst.x("scale", None)
         if scale_factor == "stack":
-            # compute the scale factor and round to nearest 10
-            scale_factor = get_stack_integral() / h.sum().value
-            scale_factor = max(round(scale_factor, -1), 1)
+            # compute the scale factor and round
+            scale_factor = round_dynamic(get_stack_integral() / h.sum().value)
         if try_int(scale_factor):
             scale_factor = int(scale_factor)
             hists[proc_inst] = h * scale_factor
-            proc_inst.label += f" x{scale_factor}"
+            proc_inst.label = inject_label(
+                proc_inst.label,
+                rf"$\times${scale_factor}",
+                placeholder="SCALE",
+            )
```

Comment:
After switching the default, you should probably pass this argument as well.
```suggestion
            proc_inst.label = inject_label(
                proc_inst.label,
                rf"$\times${scale_factor}",
                placeholder="SCALE",
                before_parentheses=True,
            )
```